### PR TITLE
Add android-ci bucket for gubernator.

### PR DESCRIPTION
**Pull request by vkryachko**
For use by github.com/firebase/firebase-android-sdk

### CODE REVIEW

**Comment by vkryachko** [comment]
cla setup

File: gcsweb.k8s.io/deployment.yaml [hunk 1]
```diff
@@ -27,6 +27,7 @@ spec:
             - -b=kubernetes-jenkins
             - -b=kubernetes-release
             - -b=kubernetes-release-dev
+            - -b=android-ci
```
Comment (ixdy) [inline gcsweb.k8s.io/deployment.yaml]: nit: I think this list is alphabetized, can you please move it before `-b=crreleases`?

Comment (vkryachko) [reply gcsweb.k8s.io/deployment.yaml]: Done

**Comment by ixdy** [comment]
deployed: http://gcsweb.k8s.io/gcs/android-ci/

**Comment by vkryachko** [comment]
Sweet, thanks!

**Pull request merged**